tests/stub/routing/scripts/v3/reader_tx_with_unexpected_interruption_on_second_run.script

```diff
@@ -0,0 +1,19 @@
+!: BOLT #VERSION#
+
+A: HELLO {"{}": "*"}
+*: RESET
+C: BEGIN {"mode": "r"}
+S: SUCCESS {}
+C: RUN "RETURN 1 as n" {} {}
+S: SUCCESS {"fields": ["n"]}
+C: PULL_ALL
+S: RECORD [1]
+   SUCCESS {"type": "r"}
+C: COMMIT
+S: SUCCESS {}
+*: RESET
+C: BEGIN {"mode": "r"}
+S: SUCCESS {}
+C: RUN "RETURN 1 as n" {} {}
+S: <EXIT>
+?: GOODBYE
```

Comment:
```suggestion
```

This line cannot be reached. After `S: <EXIT>` the server ends the script and closes the current connection.